### ISSUE TITLE
train and inference using apollo dataset

### ISSUE THREAD

**Issue by 940023303** [traveller59/second.pytorch#270]
@traveller59  I want to train the model using Apollo dataset and inference my own dataset with trained model. now I have modified some code and the process of training seems correct, but I don't know how to modify the evalution function in dataset class. Apollo dataset and my own dataset just have pointcloud and label include type and dimension and location and heading. I inference  the kitti dataset using the  model  trained with apolo dataset. This is the result.
Evaluation official
Car AP(Average Precision)@0.70, 0.70, 0.70:
bbox AP:33.25, 24.56, 21.84
bev  AP:37.20, 26.45, 21.42
3d   AP:9.70, 9.09, 9.09
aos  AP:16.76, 12.39, 11.04
Car AP(Average Precision)@0.70, 0.50, 0.50:
bbox AP:33.25, 24.56, 21.84
bev  AP:55.23, 39.50, 33.70
3d   AP:50.94, 36.79, 30.99
aos  AP:16.76, 12.39, 11.04

Evaluation coco
Car coco AP@0.50:0.05:0.95:
bbox AP:29.07, 21.98, 19.82
bev  AP:29.14, 21.61, 18.36
3d   AP:16.31, 12.79, 11.27
aos  AP:14.63, 11.09, 10.02

this is the url of downloading the apollo dataset. http://apolloscape.auto/tracking.html

Can you help me or give me some advise?

**Comment by AddASecond** [comment]
Same question. Have you solved it yet? @940023303

**Comment by 940023303** [comment]
@AddASecond  yes，i write a new function to evaluate the detection performance.

**Comment by AddASecond** [comment]
> @AddASecond yes，i write a new function to evaluate the detection performance.

How much time did you train your model? I only got 14AP in Car, using one P5000(only can batchsize 1~2) for 2~3 days.

**Comment by 940023303** [comment]
> How much time did you train your model? For Apollo, I only got 14AP in Car, using one P5000(only can batchsize 1) for 3 days.
@AddASecond  mybe six hours with 1080Ti. the average iou is near 50%.

**Comment by lonlonago** [comment]
@940023303  what‘s your ap like? could you share the evau code?

**Comment by ryontang** [comment]
@AddASecond Hi, Dose it have some differences between training by kitti data and apollo data.

**Comment by CaptainCSD** [comment]
> > @AddASecond yes，i write a new function to evaluate the detection performance.
> 
> How much time did you train your model? For Apollo, I only got 14AP in Car, using one P5000(only can batchsize 1) for 3 days.

 wtf man, 3days? I am using GTX 3080 for training with setting batch size of 6, any idea of how long it will take to finish the training ?
